### DISCUSSION TITLE
Gateway Bticino e Home Assistant

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#1712] [GENERALE - Discussioni generiche]
Ciao a tutti,
sto realizzando un impianto nella mia casa tramite bticino smart quindi deviatori connessi prese etc etc e modulo gateway bticino ma volevo qualcosa di più personalizzato e versatile dove poter gestire più scenari, collegare ulteriori dispositivi non supportati da bticino e programmare azioni.
leggendo le varie guide sono orientato per installare un Raspebby e qui inizia il mio dilemma.... la logica sarebbe montare ConBee II e associare tutto bypassando il gateway bticino ma cosi mi perderei agg. firmware eventuali dispositivi e in più dal momento che l'implementazione di HA avverrà pian piano e in casa ci abito, vorrei se posse possibile la "convivenza" dell'app Bticino tramite gateway proprietario e la possibilità di usare anche HA.
Ringrazio tutti in anticipo!

**Comment by esposimo** [comment]
Ciao,

per quanto ne so io, se associ i dispositivi al gateway , HA non vede più nulla.
Se però usi la conBee dovresti vedere tutti i deviatori etc..ma con qualche limitazione (forse). 
Qui trovi i dispositivi supportati e cosa espongono quando non usi il gateway https://www.zigbee2mqtt.io/supported-devices/#v=BTicino

P.S. il gateway è comunque fondamentale per una prima associazione, credo. Spero qualcuno possa darti info maggiori, perchè questo è quello che ho capito io informandomi sulla serie bticino

**Comment by Dim-1** [comment]
Mi ero interessato anche io in passato a questa cosa, tanto che metterò pure io dei componenti livinglight connessi e gateway. Avevo letto nel web che qualcuno aveva integrato i moduli BTicino, compreso gateway, con il componente homekit controller, che integra i dispositivi integrabili in homekit di Apple. I dettagli non li so, ancora non ho fatto niente personalmente.
